js/main.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+function start(canvas) {
+  const game = new Game(canvas.getGridDimensions().width, canvas.getGridDimensions().height);
+
+  game.init();
+  game.draw(canvas);
+
+  function frame() {
+    canvas.clear();
```

Comment:
You should clear the canvas right before drawing the new frame (otherwise the canvas will be blank while the game updates).